src/test/ai-accuracy-tests/ai-backend.ts

```diff
@@ -0,0 +1,158 @@
+import OpenAI from 'openai';
+import type { ChatCompletionCreateParamsBase } from 'openai/resources/chat/completions';
+import Anthropic from '@anthropic-ai/sdk';
+import type { TextBlock } from '@anthropic-ai/sdk/resources';
+
+import { CHAT_PARTICIPANT_MODEL } from '../../participant/constants';
+
+let anthropic: Anthropic;
+if (process.env.ANTHROPIC_API_KEY) {
+  anthropic = new Anthropic({
+    apiKey: process.env.ANTHROPIC_API_KEY,
+  });
+}
```

Comment:
Do we need lines 8-13 when it looks like `getAnthropicClient()` does the same?

Reply:
Ah yeah we don't need those, removed. Thanks